Add JSON key content for auth (#68)
logstash-plugins/logstash-input-google_pubsub: README.md

1. If you are running Logstash outside of Google Cloud Platform, then you will
need to create a Google Cloud Platform Service Account and specify the full

> path to the JSON private key file in your config in `json_key_file` attribute.
> If you do not want to upload file and want to accept it as a string, then add the content 
> of JSON private key in `json_key_file_content` attribute. You must assign sufficient

Reviewer: ```suggestion
path to the JSON private key file in your config in `json_key_file` setting.
If you don't want to upload the file file and use its content as a string, then add the content 
of JSON private key in `json_key_file_content` setting. You must assign sufficient
```

Author: @andsel Thanks for the suggestion I have updated the doc in README and index.asciidoc as per your suggestion.
